pallets/phala/src/registry.rs

```diff
@@ -337,6 +346,42 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// (called by a prouter on behalf of a worker)
+		#[pallet::weight(0)]
+		pub fn bind_worker_pnetwork_ident(
```

Comment:
I have an idea. We don't need to assume the network endpoint must to be i2p. I think any valid http endpoint can be accepted. So theoretically if a user have to publish his public accessible endpoint address (http://ip:port), it's also acceptable.

So I'd suggest to rename this function to "bind_worker_endpoint", not mentioning pnetwork (actual we have to avoid "pnetwork" anyway because is already widely used by another project)